.github/workflows/ci.yaml

```diff
@@ -124,6 +124,10 @@ jobs:
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       AIRFLOW__ASTRO_SDK__SQL_SCHEMA: astroflow_ci
+      REDSHIFT_DATABASE: dev
+      REDSHIFT_HOST: utkarsh-cluster.cdru7mxqmtyx.us-east-2.redshift.amazonaws.com
```

Comment:
Should this URL be part of repo? It can be taken from secrets?

Reply:
I was planning to keep it as part of secrets earlier. But, Kaxil mentioned that it is not a secret and can be taken as an environment variable.